### ISSUE TITLE
[Ruby] Incorrect highlighting when use '/'

### ISSUE THREAD

**Issue by lanrion** [sublimehq/Packages#639]
## Environment
- Operating system and version: Mac OS X 10.11.6
-  Sublime Text: Build 3124

<img width="865" alt="ruby-wrong" src="https://cloud.githubusercontent.com/assets/1432529/18805221/eeb4f2c0-8239-11e6-9fe9-955225dc443c.png">

**Comment by wbond** [comment]
Please provide the raw code, so I can copy and paste

**Comment by lanrion** [comment]
@wbond 

```
def self.index_to_column_char(index)
  str = 'A'
  if index >= 26*27
    str = "#{('A'.ord + index / (26*26) - 1).chr}"
    index = index%(26*26)
    str += "#{('A'.ord + index / 26 - 1).chr}#{('A'.ord + index%26).chr}"
  elsif index >= 26
    str = "#{('A'.ord + index / 26 - 1).chr}#{('A'.ord + index%26).chr}"
  else
    str = ('A'.ord + index).chr
  end
  return str
end
```

**Comment by wbond** [comment]
This is fixed by 458e0764063a885cf8f7c2da3b41ac3d81fdfa57, which will be part of the next dev build
